### DISCUSSION TITLE
Welcome to openai-translator Discussions!

### DISCUSSION THREAD

**Discussion by yetone** [nextai-translator/nextai-translator#57] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 1ifebytes** [comment]
不知道我设置不对，还是哪里有问题，安装完成后就没有打开过。。
![image](https://github.com/openai-translator/openai-translator/assets/76526330/c156622f-2193-4f9d-9010-df08eed282ad)
如下是本机器配置
<img width="268" alt="image" src="https://github.com/openai-translator/openai-translator/assets/76526330/6dde418c-2bab-465b-ab75-247c8a40014b">

**Comment by DKksw** [comment]
Window 下载 OpenAI Traanslator之后无法完成翻译，已配置API Key 但是翻译显示报错： You exceeded your current quota, please check your plan and billing details. For more information on this error, read the docs: https://platform.openai.com/docs/guides/error-codes/api-errors. 我的GPT账号支持4.0，现在在OpenAI Traanslator用3.5也不行。

**Reply by DKksw** [reply]
![screenshot-20240225-174217](https://github.com/openai-translator/openai-translator/assets/56796442/6e44817c-0cb0-481a-9ef2-8327568fcbb7)

**Reply by CrazyPoohBear** [reply]
我遇到同樣問題+1

**Comment by chengfengfengwang** [comment]
> ![screenshot-20240225-174217](https://private-user-images.githubusercontent.com/56796442/307627147-6e44817c-0cb0-481a-9ef2-8327568fcbb7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQwNDQ0OTksIm5iZiI6MTcxNDA0NDE5OSwicGF0aCI6Ii81Njc5NjQ0Mi8zMDc2MjcxNDctNmU0NDgxN2MtMGNiMC00ODFhLTllZjItODMyNzU2OGZjYmI3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI1VDExMjMxOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdjYTBlY2ZjMzE3MGE3ZDBlZGY5MTkyZWI1ZWFhN2FkYzNiYTJmOTI4MDBkMGUxY2Q1NGE2NzEyN2NlMjVjYmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.mkY738_DRfEUhs9Gjh6dhQmv8ZHhUZyG758FM0XkAbY)

在chatgpt网站可以用和调api不是一回事，前者可以看成是订阅制，后者是按量付费
